renovation_core_graphql/tags/types/root.graphql

```diff
@@ -0,0 +1,25 @@
+input TagAddInput {
+    doctype: String!
+    docname: String!
+    tag: String!
+}
+
+input TagRemoveInput {
+    doctype: String!
+    docname: String!
+    tag: String!
+}
+
+input TagFilterInput{
+    doctype: String
+    search: String
+}
+
+extend type Query {
+    tags(filters: TagFilterInput, limitStart: Int, limitPageLength: Int): [String!]
```

Comment:
add docs by default 10 is returned ?
or should we force user to send limitStart and limitPageLength..?
also what say if limitPageLength is 0 send all ?

Reply:
I'll add the docs. 

For the `limitPageLength` as 0, I think better not to return all. The user has always to specify or defaults to 10.

Reply:
Done